directives/questionnaire.py

```diff
@@ -124,6 +123,13 @@ def run(self):
                 raise SphinxError(u'Number of fields to sample randomly should greater than zero.')
             data[u'fieldgroups'][0]['pick_randomly'] = pick_randomly
 
+        if 'title' in self.options:
+            data['title'] = self.options.get('title')
+        else:
+            data[u'title|i18n'] = translations.opt('feedback') if is_feedback else translations.opt('exercise', postfix=u" {}".format(key))
+
+        self.set_assistant_permissions(data)
```

Comment:
I would prefer that the title is handled before the override if statement since the override setting is supposed to be able to override any configuration values. 

Additionally, the set assistant permissions method must not be duplicated here.